cypress/support/step_definitions/genericSteps/genericSteps.ts

```diff
@@ -79,6 +79,11 @@ Then('I see {string} is {string}', (accountList: string, value: string) => {
 When('I reload the page', () => {
   cy.reload();
 });
+
+Then('I see {string} link', (buttonName: string) => {
+  cy.contains('a', buttonName).should('exist', buttonName);
+
```

Comment:
can this be moved to the genericSteps/linkSteps.ts please?
Also the regex should be 'I see **the** {string} link

Reply:
okay

Reply:
already in genericsteps.ts

Reply:
yep, so just needs to be moved from **_genericSteps.ts_** _**to genericSteps/linkSteps.ts**_

Reply:
![Uploading image.png…]()

Reply:
deleted this step in the last commit